config/clusters/utoronto/common.values.yaml

```diff
@@ -36,6 +36,16 @@ jupyterhub:
           name: University of Toronto
           url: https://www.utoronto.ca/
         announcements:
+          - |
+            <div>
+              <strong>ARC will be making changes to our educational JupyterHub on January 9, 2023</strong>: R/RStudio will be
```

Comment:
Let's turn this into an `<h4>` to match what is in the prior announcement? We can then put the other two paragraphs inside `<p>` tags so it looks ok. Looks good to merge after those changes.